### PR TITLE
enhancement(templates): use latest faas-js-runtime + cloudevents

### PR DESCRIPTION
**Pull request by lance**
Better typing of handle functions and return values for typescript based functions.

/kind enhancement

Signed-off-by: Lance Ball <lball@redhat.com>

```release-notes
bumps faas-js-runtime to 0.9.5 using latest cloudevents and type definitions in Node.js and TypeScript functions
```

### CODE REVIEW

File: templates/typescript/cloudevents/src/index.ts [hunk 1]
```diff
@@ -44,7 +47,12 @@ ${JSON.stringify(cloudevent.data)}
 -----------------------------------------------------------
 `);
   // respond with a new CloudEvent
-  return HTTP.binary(new CloudEvent({ ...meta, data: cloudevent.data }));
+  return new CloudEvent<Customer>({ ...meta, data: cloudevent.data as Customer });
```
Comment (matejvasek) [inline templates/typescript/cloudevents/src/index.ts:50]: Wouldn't `cloudevent!.data` work instead of `cloudevent.data as Customer`?

**Comment by lance** [comment]
@matejvasek ptal

**Comment by matejvasek** [comment]
I see `FUNC_LOG_LEVEL` at 3 places not 4, why?

**Comment by lance** [comment]
> I see `FUNC_LOG_LEVEL` at 3 places not 4, why?

Oversight. I've taken care of it.

**Pull request merged**